mix.exs

```diff
@@ -33,7 +33,10 @@ defmodule Membrane.MP4.Plugin.MixProject do
       main: "readme",
       extras: ["README.md"],
       source_ref: "v#{@version}",
-      nest_modules_by_prefix: [Membrane.MP4]
+      nest_modules_by_prefix: [Membrane.MP4],
+      groups_for_modules: [
+        Muxers: [Membrane.MP4.Muxer.CMAF, Membrane.MP4.Muxer.ISOM]
+      ]
```

Comment:
```suggestion
      nest_modules_by_prefix: [Membrane.MP4,  Membrane.MP4.Muxer, Membrane.MP4.Payloader],
      groups_for_modules: [
        Muxers: ~r/Membrane.MP4.Muxer/,
        Payloaders: ~r/Membrane.MP4.Payloader/
      ]
```

This suggestion requires a few more changes - I'd move atom-related modules to `Membrane.MP4`, e.g. `Membrane.MP4.MovieBox` etc.